platform-sdk/swirlds-base/src/testFixtures/java/module-info.java

```diff
@@ -2,5 +2,5 @@
     exports com.swirlds.base.test.fixtures.time;
 
     requires transitive com.swirlds.base;
-    requires com.swirlds.common;
+    requires com.swirlds.common; // Should be removed in future
```

Comment:
could you please create a ticket for that explaining why exactly it should be removed?

Reply:
I just moved the comment. It was in build.gradle.kts before:

https://github.com/hashgraph/hedera-services/blob/7c1b84274c1a8157b70744700a8d4a27579e901e/platform-sdk/swirlds-base/build.gradle.kts#L30-L31

Unfortunately, I do not know the background of why it was there in the first place. I just did not want to remove it silently.

Should I remove it?